spec/classes/test_duplicate_alias_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
+describe 'test::duplicate_alias' do
+  it { should compile }
```

Comment:
should you add positive and negative tests here that `foo_bar_1`, and `foo_bar_2` can be found, but not `/bin/echo foo bar`?